sklearn/model_selection/tests/test_search.py

```diff
@@ -1199,6 +1200,72 @@ def compare_refit_methods_when_refit_with_acc(search_multi, search_acc, refit):
         assert getattr(search_multi, key) == getattr(search_acc, key)
 
 
+@pytest.mark.filterwarnings("ignore:The parameter 'iid' is deprecated")  # 0.24
+@pytest.mark.parametrize('search_cv', [RandomizedSearchCV, GridSearchCV])
```

Comment:
We can create the objects here:

```py
@pytest.mark.parametrize('search_cv', 
    [RandomizedSearchCV(estimator=..., param_distributions=...)
```

And we will not need the `if` statement in the test.